package.json

```diff
@@ -17,7 +17,8 @@
     "precise-commits": "precise-commits",
     "prepublishOnly": "npm run build",
     "start": "node index.js",
-    "test": "npm run lint",
+    "test": "mocha --timeout=20000 --bail --exit --recursive ./test/**/*.spec.js",
```

Comment:
Let's not remove our lint command. Having no linting errors before we continue is important for keeping consistent coding patterns.

Reply:
okay I'll add a new lint command since running linting inside a test command is confusing

Reply:
it can be done under one command, or we can tweak the CI, not a big deal either way, but if this is changed or removed, we should make a notable effort to ensure that we continue to lint

Reply:
We have lint command we can run it throw Travis cuz ruining linting inside test command is confusing 
` "lint": "eslint --config .eslintrc.json ./",`